### DISCUSSION TITLE
Android app crashed when click to subpages

### DISCUSSION THREAD

**Discussion by WJ-Yuan** [community/community#50875] [Mobile]
### Select Topic Area

Bug

### Body

**Device**
Xiaomi Mi 10
**System**
MIUI 13.0.7
**Android Version**
12

## Detail
When I click to subpages, the app crash back to the main page.

![GIF 2023-3-24 9-34-25](https://user-images.githubusercontent.com/59187817/227402080-da4dc324-3f6f-491d-899d-f775b93d4746.gif)

**Comment by aconsuegra** [comment]
Thanks for your feedback @WJ-Yuan. Are you using the latest version of the app available in Google Play?

**Reply by WJ-Yuan** [reply]
> Thanks for your feedback @WJ-Yuan. Are you using the latest version of the app available in Google Play?

yes, the app version is 1.105.0

**Reply by aconsuegra** [reply]
Thanks for the quick reply! We'll investigate what is going on.

**Reply by MinaEliaa** [reply]
what is the problem

**Comment by WJ-Yuan** [comment]
I updated version to 1.111.1 today, there is no crash event anymore. tyty.
